common/data/okn/okn.ts

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable class-methods-use-this */
+import { fetchAPI, getObjectsTotalCount, STRAPI_BASE_URL } from '../dataHelpers';
+import { canGetById } from '../base/canGetById';
+import { Area } from '../base/objectsBase';
+import { OknAreaType, OknObjectSignificanceType } from './oknConstants';
+import { OknObject, OknObjectWithGeometry } from './oknObject';
+
+export const okn = {
+    async getObject(id: string): Promise<OknObject> {
+        return canGetById.getObject(id, '/okn-objects')
+            .then((x) => {
+                // eslint-disable-next-line no-param-reassign
+                x.attributes.img = x.attributes?.img.split("','")[0].slice(8);
```

Comment:
Can we just replace single quotes to double quotes, parse json and get image?`JSON.parse(x.attributes.img.replaceAll("'", '"'))`